registry/index.adoc

```diff
@@ -0,0 +1,28 @@
+[id="registry-overview"]
+= {product-title} registry overview
+include::modules/common-attributes.adoc[]
+:context: registry-overview
+
+toc::[]
+
+{product-title} can build images from your source code, deploy them, and manage their lifecycle. It provides an internal, integrated container image registry to enable this that can be deployed in your {product-title} environment to locally manage images. This overview contains reference information and links for registries commonly used with {product-title}, with a focus on the internal image registry.
```

Comment:
The phrase "to enable this" seems unnecessary. Consider removing.

Reply:
acked